src/entity/Alerts.ts

```diff
@@ -60,6 +60,9 @@ export class Alerts {
 
   bootPopup?: boolean;
 
+  @Column({ type: 'uuid', nullable: true })
+  topReaderBadge?: string | null;
```

Comment:
@capJavert I wanted to make this reference the actual ID in the database, but not sure what relation to use with typeorm. Or should I just add it as a manual query?

Reply:
My reasoning is to put the `id` of `user_top_reader` into alerts, then in frontend I can just use the `topReaderBadge` query based on that ID to fetch it, instead of `topReader` on user and then pick the latest

Reply:
Relation would be OneToOne with onDelete: `SET NULL`, basically FK if I see correctly. 

> Or should I just add it as a manual query?

Since user is fetched on most places I would probably lean to just doing a query, because else we need to maintain and update this alert which gives an opportunity of it getting out of sync. When we take the latest one we don't have to fetch anything else, we just show the data?

Reply:
> Relation would be OneToOne with onDelete: SET NULL, basically FK if I see correctly.

I tried that, but the migration maker didn't output anything for it 😬

Reply:
But I can change alert to be boolean, and then use the whoami query with topReader selector, but update it to accept a limit, so we don't need to fetch 5, when we only want the latest one anyway

Reply:
Yeah, boolean would be more simple, when would we remove the alert then, when users sees the popup?

Reply:
When the user closes the modal